Presentation.Web/Controllers/API/V2/External/Tokens/TokenV2Controller.cs

```diff
@@ -20,7 +21,7 @@ public class TokenV2Controller : ExternalBaseController
         [IgnoreCSRFProtection]
         public IHttpActionResult Introspect([FromBody] TokenIntrospectionRequest request)
         {
-            return new TokenValidator().VerifyToken(request.Token)
+            return new TokenValidator(Settings.Default.BaseUrl).VerifyToken(request.Token)
```

Comment:
Inject the ITokenValidator in the constructor (after injecting the URL in the builder